src/js/containers/recipient/RecipientContainer.jsx

```diff
@@ -109,21 +107,20 @@ const RecipientContainer = (props) => {
     }, []);
 
     useEffect(() => {
-        if (prevProps) {
-            if (recipientId !== prevProps.recipientId) {
-                // Reset the FY
-                props.setRecipientFiscalYear(fy);
-                loadRecipientOverview(recipientId, defaultFy);
-            }
-            if (prevProps.fy !== fy) {
-                props.setRecipientFiscalYear(fy);
-            }
-            if (props.recipient.fy !== prevProps.recipient.fy) {
-                loadRecipientOverview(recipientId, props.recipient.fy);
-            }
-        }
-        /* eslint-disable-next-line react-hooks/exhaustive-deps */
-    }, [recipientId, fy]);
+        props.setRecipientFiscalYear(fy);
+        loadRecipientOverview(recipientId, defaultFy);
```

Comment:
should loadRecipientOverview(recipientId, defaultFy) go in the useEffect with fy as the dependency?  I noticed some weird behavior.  the backend is deploying so will try to reproduce later